docs/visualize/aggregations.asciidoc

```diff
@@ -1,36 +1,51 @@
 
-{ref}/search-aggregations-metrics-percentile-aggregation.html[Median]:: The value that is in the 50% percentile.
+| {ref}/search-aggregations-metrics-max-aggregation.html[`Max`]
+ | The highest value.
 
-{ref}/search-aggregations-metrics-min-aggregation.html[Min]:: The lowest value.
+| {ref}/search-aggregations-metrics-percentile-aggregation.html[`Median`]
+ | The value that is in the 50% percentile.
 
-{ref}/search-aggregations-metrics-percentile-rank-aggregation.html[Percentile ranks]:: Returns the percentile rankings for the values in the specified numeric field. Select a numeric field from the drop-down, then specify one or more percentile rank values in the *Values* fields.
+| {ref}/search-aggregations-metrics-min-aggregation.html[`Min`]
+ | The lowest value.
 
-{ref}/search-aggregations-metrics-percentile-aggregation.html[Percentiles]:: Divides the
-values in a numeric field into specified percentile bands. Select a field from the drop-down, then specify one or more ranges in the *Percentiles* fields.
+| {ref}/search-aggregations-metrics-percentile-rank-aggregation.html[`Percentile ranks`]
+ | Returns the percentile rankings for the values in the specified numeric field. Select a numeric field from the drop-down, then specify one or more percentile rank values in the *Values* fields.
 
-Standard Deviation:: Requires a numeric field. Uses the {ref}/search-aggregations-metrics-extendedstats-aggregation.html[_extended stats_] aggregation.
+| {ref}/search-aggregations-metrics-percentile-aggregation.html[`Percentiles`]
+ | Divides the values in a numeric field into specified percentile bands. Select a field from the drop-down, then specify one or more ranges in the *Percentiles* fields.
 
-{ref}/search-aggregations-metrics-sum-aggregation.html[Sum]:: The total value.
+| `Standard Deviation`
+ | Requires a numeric field. Uses the {ref}/search-aggregations-metrics-extendedstats-aggregation.html[_extended stats_] aggregation.
 
-{ref}/search-aggregations-metrics-top-hits-aggregation.html[Top hit]:: Returns a sample of individual documents. When the Top Hit aggregation is matched to more than one document, you must choose a technique for combining the values. Techniques include average, minimum, maximum, and sum.
+| {ref}/search-aggregations-metrics-sum-aggregation.html[`Sum`]
+ | The total value.
 
-Unique Count:: The {ref}/search-aggregations-metrics-cardinality-aggregation.html[Cardinality] of the field within the bucket.
+| {ref}/search-aggregations-metrics-top-hits-aggregation.html[`Top hit`]
+ | Returns a sample of individual documents. When the Top Hit aggregation is matched to more than one document, you must choose a technique for combining the values. Techniques include average, minimum, maximum, and sum.
+
+| `Unique Count`
+ | The {ref}/search-aggregations-metrics-cardinality-aggregation.html[Cardinality] of the field within the bucket.
```

Comment:
Is there a link for Unique Count?  Also, I think cardinality should be lower case.  Also "script value source"

Reply:
Unique Count is unique to Kibana.